src/components/Coaching/CoachingDetail/AppointmentResults/AppointmentResults.tsx

```diff
@@ -79,14 +77,7 @@ export const AppointmentResults: React.FC<AppointmentResultsProps> = ({
 
   return (
     <AnimatedCard>
-      <CardHeader
-        title={
-          <Box display="flex" alignItems="center">
-            <Box flex={1}>{t('Appointments and Results')}</Box>
-            <HelpButton articleVar="HS_COACHING_APPOINTMENTS_AND_RESULTS" />
```

Comment:
That is a shame we're losing this functionailty. Can can't link to the HelpJiuce page?

Reply:
Thanks for calling that out. I don't have article URLs yet, but I added back functionality to make these help buttons link to a particular URL.